packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -440,15 +517,17 @@ export class TokenRatesController extends BaseController<
    * If not supported, it fetches contractExchange rates and maps them from token/fallback-currency
    * to token/nativeCurrency.
    *
-   * @param nativeCurrency - The native currency of the currently active network.
-   * @param slug - The unique slug used to id the chain by the coingecko api
+   * @param nativeCurrency - The ticker for the chain.
+   * @param chainSlug - The unique slug used to id the chain by the coingecko api
    * should be used to query token exchange rates.
+   * @param tokenAddresses - The addresses for the token contracts.
    * @returns An object with conversion rates for each token
    * related to the network's native currency.
    */
   async fetchAndMapExchangeRates(
```

Comment:
Can this be made private? I'm not seeing it used by either of the clients or other controllers?

Reply:
typescript wouldn't be happy with that since these methods are tested directly in spec

Reply:
I'm not seeing this one in the test file?

Reply:
Ok lets make this private in a subsequent PR per @mcmire 's[ suggestion](https://consensys.slack.com/archives/C063W9EE484/p1700166880644369)